prevents raw redis and memcached compression (#39728)
rails/rails: activesupport/CHANGELOG.md

> *   Prevents `RedisCacheStore#read` and `MemCacheStore#read` from performing compression when `raw: true`. Compression
>     should only be performed on cache writes, not reads.

Reviewer: ```suggestion
*   Prevent `RedisCacheStore` and `MemCacheStore` from performing compression
    when reading entries written with `raw: true`.
```
